adaptors/library/jobs/generate-pdf.js

```diff
@@ -0,0 +1,19 @@
+// Generate a PDF document from a HTML string
+
+fn(state => {
+  const { data } = state;
+
+  state.pdfHTMLContent = `<html>
+       <body style="font-family: Arial, sans-serif; font-size: 14px;">
+         <h1>Sales Report</h1>
+         <p>Date: ${data.date}</p>
+         <p>Total Sales: $${data.totalSales}</p>
+      </body>
+    </html>`;
+
+  return state;
+});
+
+generatePDF($.pdfHTMLContent, {
```

Comment:
Can we add an example of what you might do with the pdf once generated? Like upload it with request?